pkg/internal/codegen/parse/crd.go

```diff
@@ -113,7 +113,13 @@ func (b *APIs) parseCRDs() {
 							resource.CRD.Spec.Subresources.Scale.LabelSelectorPath = &labelSelctor
 						}
 					}
-
+					if IsPrintColumn(resource.Type) {
```

Comment:
Can we rename `IsPrintColumn` to `hasPrintColumns` to keep it consistent with other annotations above ?